contracts/README.md

```diff
@@ -0,0 +1,24 @@
+### Build
+
+```shell
+$ forge build
+```
+
+### Generate bindings 
+
+```shell
+$ forge bind --crate-name pod-contracts --bindings-path ./bindings --alloy-version v0.9.2 --overwrite 
+```
+
+### Format
+
+```shell
+$ forge fmt
+```
+
+### Deploy
+
+```shell
+$ forge script script/Counter.s.sol:CounterScript --rpc-url <your_rpc_url> --private-key <your_private_key>
```

Comment:
The counter contract seems to be missing, which is causing to CI to fail too

Reply:
yaah thats the default contract provided by the setup. i removed it. will fix the ci only to build and generate bindings